test/e2e/cluster_config.sh

```diff
@@ -35,7 +35,7 @@ clusters:
     nlb_switch: "pre"
     vm_dirty_bytes: 134217728
     vm_dirty_background_bytes: 67108864
-    prometheus_tsdb_retention_size: enabled
+    prometheus_retention_size: 1258292KB
```

Comment:
This is the value that would have been used for test clusters, the only user of this config item.

Reply:
Changed to the more readable 1280MB.